src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+@import './utils/variables';
+@import './utils/mixins';
+
 body {
   margin: 0;
 }
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  $duration: 60s;
+
+  width: $watch-size;
+  height: $watch-size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__minute {
+    width: $minute-w;
+    height: $minute-h;
```

Comment:
The variables `$minute-w` and `$minute-h` are used here. Ensure these variables are defined in the imported files or within this file to avoid compilation errors.